contracts/governance/GraphGovernance.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.3;
+
+import "../upgrades/GraphUpgradeable.sol";
+
+import "./GraphGovernanceStorage.sol";
+
+/**
+ * @title Graph Governance Contract
+ * @notice Governance contract used to inscribe Graph Council and community votes.
+ */
+contract GraphGovernance is GraphGovernanceV1Storage, GraphUpgradeable, IGraphGovernance {
+    // -- Events --
+
+    event ProposalCreated(
+        address submitter,
```

Comment:
This will always be the governor - i.e. the graph council multisig. 

do we care to emit this every time?

Reply:
Probably we can remove it, we can read who is the submitter by reading the contract governor

Reply:
ill remove it